paver.sh

```diff
@@ -99,6 +99,12 @@ clean_build() {
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
     rm -rf /tmp/pip*
+
+    # On the OSX build server tmp is in $TMPDIR
+    # check if TMPDIR is set before trying to clean it.
+    if [ ! -z "${TMPDIR-}" ]; then
+        rm -rf ${TMPDIR}/pip*
+    fi
```

Comment:
maybe and an else here and put the above ` rm -rf /tmp/pip*` ... so that when we have TMPDIR we only delete TMPDIR and not other folders